### PR TITLE
LPS-55098

### PR DESCRIPTION
**Pull request by jonathanmccann**
/cc @arbetts

### CODE REVIEW

**Comment by jonathanmccann** [comment]
Hey Eduardo,

Please see the relevant discussion on the LPS for more information about this fix.

Thanks!

**Comment by epgarcia** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by epgarcia** [comment]
Hi @jonathanmccann,

After reviewing both the discussion in the LPS and the changes I have some comments:

* The fix focuses in the case of browsing assets from the web content configuration, but what happens with the manual selection in Asset Publisher or the Related Links selection? They suffer the same issue, right?
* Must we really force the use of DB in asset browser? I think we should find a solution for both index and database queries that involve ignoring searchability.
* I don't like the concept of adminSearch as a param to the AssetEntryService. Someone who uses the API cannot tell what this involves and if it's related with the admin role. I'd rather use more explicit parameters like includeSearchable, includeListable

In summary, I think we need a solution that covers all cases (not only web content display configuration) and all search systems (DB and Index). I'll start a conversation about this to find the best solution.

Thanks!
